pkg/resources/collector/templates.go

```diff
@@ -230,7 +230,12 @@ processors:
       - key: tier
       - key: component
     {{ template "metrics-system-processor" . }}
+extensions:
+  file_storage:
+    directory: /etc/otel/filestorage
```

Comment:
proper convention to use `/var/<something>` for stateful data

Reply:
Changed to `/var/otel/filestorage`. Could also be `/var/log/otel/filestorage` but I think `/var/log` is more prone to manual deleting, if that makes sense.